vtex/toolbelt
# Use prompts package for prompts

#### What is the purpose of this pull request?
The `prompts-confirm` package had some bugs and is currently unmaintained. It migrated to inside `enquirer`, but it has some bugs as well that became the migration unviable.
`prompts` package seems to solve the problem, but slightly changes the developer experience:
```
Answering confirmation prompts now only need the confirmation character (or just enter). No need to type the characted and enter. 
```

#### How should this be manually tested?
```
cd /tmp && rm -rf toolbelt && \
git clone https://github.com/vtex/toolbelt.git && \
cd toolbelt && \
git checkout fix/use-prompts-package-for-prompts && \
yarn && yarn build && yarn global add file:$PWD
```
Try `vtex login`, which has a prompt.
Check if pressing <kbd>Ctrl</kbd>+<kbd>C</kbd> during/after the prompt closes the process, returns to the shell and has the usual terminal cursor.


#### Types of changes

#### Chores checklist
